optimade/adapters/structures/proteindatabank.py

```diff
@@ -0,0 +1,241 @@
+        # "_atom_site.label_seq_id\n"
+        "_atom_site.occupancy\n"  # species.concentration
+        f"_atom_site.{coord_type}_x\n"  # cartesian_site_positions
+        f"_atom_site.{coord_type}_y\n"  # cartesian_site_positions
+        f"_atom_site.{coord_type}_z\n"  # cartesian_site_positions
+        "_atom_site.thermal_displace_type\n"  # Set to 'Biso'
+        "_atom_site.B_iso_or_equiv\n"  # Set to 1.0:f
+    )
+
+    if coord_type == "fract":
+        sites = attributes.fractional_site_positions
+    else:
+        sites = attributes.cartesian_site_positions
+
+    species: Dict[str, OptimadeStructureSpecies] = {
+        species.name: species for species in attributes.species
+    }
+
+    for site_number in range(attributes.nsites):
+        species_name = attributes.species_at_sites[site_number]
+        site = sites[site_number]
+
+        current_species = species[species_name]
+
+        for index, symbol in enumerate(current_species.chemical_symbols):
+            if symbol == "vacancy":
+                continue
+
+            label = f"{species_name.upper()}{site_number + 1}"
+            if len(current_species.chemical_symbols) > 1:
+                if (
+                    "vacancy" in current_species.chemical_symbols
+                    and len(current_species.chemical_symbols) == 2
+                ):
+                    pass
+                else:
+                    label = f"{symbol.upper()}{index + 1}"
+
+            cif += (
+                f"ATOM  {site_number + 1:5d}  {symbol}  {label:8}  "
+                f"{current_species.concentration[index]:6.4f}  {site[0]:8.5f}  "
+                f"{site[1]:8.5f}  {site[2]:8.5f}  {'Biso':4}  {'1.000':6}\n"
+            )
+
+    return cif
+
+
+def get_pdb(  # pylint: disable=too-many-locals
+    optimade_structure: OptimadeStructure,
+) -> str:
+    """ Write Protein Data Bank (PDB) structure in the old PDB format from OPTIMADE structure
+
+    Inspired by `ase.io.proteindatabank.write_proteindatabank()` in the ASE package.
+
+    :param optimade_structure: OPTIMADE structure
+    :return: str
+    """
+    if globals().get("np", None) is None:
+        warn(NUMPY_NOT_FOUND)
+        return None
+
+    pdb = ""
+
+    attributes = optimade_structure.attributes
+
+    rotation = None
+    if any(attributes.dimension_types):
+        # Turn null/None values in lattice_vectors into float(nan)
+        lattice_vectors, _ = pad_cell(attributes.lattice_vectors, padding="nan")
+        currentcell = np.asarray(lattice_vectors)
+        cellpar = cell_to_cellpar(currentcell)
+        exportedcell = cellpar_to_cell(cellpar)
+        rotation = np.linalg.solve(currentcell, exportedcell)
+        # ignoring Z-value, using P1 since we have all atoms defined explicitly
+        pdb += (
+            f"CRYST1{cellpar[0]:9.3f}{cellpar[1]:9.3f}{cellpar[2]:9.3f}"
+            f"{cellpar[3]:7.2f}{cellpar[4]:7.2f}{cellpar[5]:7.2f} P 1\n"
+        )
+
```

Comment:
I've had some issues (again with VESTA) getting the positions read in the correct format. I think if you supply a `CRYST` block, some readers/viewers assume the positions are fractional. We can fix this by adding [`SCALEn` blocks](https://www.wwpdb.org/documentation/file-format-content/format33/sect8.html#SCALEn) that contain the transformation back to fractionals. If we're happy to use numpy here, this would be (seems to ned the extra 0):

```suggestion

         vol = np.dot(cellpar[0], np.cross(cellpar[1], cellpar[2]))
         for i, vec in enumerate(cellpar):
                scale = np.cross(cellpar[i + 1 % 3], cellpar[i + 2 % 3]) 
                pdb += (
                    f"SCALE{i} {scale[0]:9.3f}{scale[1]:9.3f}{scale[2]:9.3f} 0.00000"
                )   
```